src/base.py

```diff
@@ -19,8 +19,9 @@ def accuracy_thresh_expand(y_pred:Tensor, y_true:Tensor, thresh:float=0.5, sigmo
     return ((y_pred>thresh)==y_true[:,None].expand_as(y_pred).bool()).float().mean()
 
 def create_dmg(path_hr, path_lr, newdmg = False):
-    os.remove(path_lr/'processed.txt')
-    if len(os.listdir(path_lr)) < 0 or newdmg == True:
+    try: os.remove(path_lr/'processed.txt')
```

Comment:
The indention here is against formatting principles of python. Change to multiple lines since that is more standard.